### ISSUE TITLE
Grade

### ISSUE THREAD

**Issue by bursteinalan** [CS521-Burstein/final-project-alexmao0501#4]
Really great work on the writeup and detailed explorations! 101%

**Comment by bursteinalan** [comment]
Do I have your permission to use this paper as an example for future semesters?

**Comment by ddemaest-bu** [comment]
@bursteinalan Of course, feel free to use the paper as an example! Thank you again for a great course.
